bigquery/biglake/bigquery_create_unpartitioned_biglake_table/main.tf

```diff
@@ -81,7 +81,13 @@ resource "google_bigquery_dataset" "default" {
 
 
 # This creates a BigQuery table named "my_table" in the dataset "default".
-# The table has three columns, named "country", "product", and "price", which are of types STRING, STRING, and INT64 respectively.
+# The table has three columns, named "country", "product", and "price", which
+# are of types STRING, STRING, and INT64 respectively.
+# The table has Automatic Metadata Caching enabled.
+# https://cloud.google.com/bigquery/docs/biglake-intro#metadata_caching_for_performance
+# `metadata_cache_mode` may be set to `MANUAL` and `max_staleness` ommitted to
+# use MANUAL Metadata refresh `metadata_cache_mode` and `max_staleness` may be
+# set to omitted to disable the feature.
```

Comment:
Suggestion: There are region tags are in this file. So, if you are using this sample in the cloud docs, kindly keep code comments to be simple, short, to the point.

Reply:
done.

simplified

Reply:
done.